### DISCUSSION TITLE
iphone8+ board configuration?

### DISCUSSION THREAD

**Discussion by akyora** [airsquared/blobsaver#326] [Q&A] [answered]
The board configuration on the iphone8+ should have D21AP and D21AAP, but they are grayed out and cannot be entered.
Also, when I press the read button, it does not reflect the configuration.
Is this an oversight?
Or is this correct?
Note that tsssaver requires you to enter this information.

**Comment by airsquared** [comment, marked as answer]
The board configuration D21AAP actually doesn't exist, it seems that whoever put it into the iPhone Wiki was mistaken. Apple doesn't have any iOS versions for the board configuration D21AAP.

**Reply by akyora** [reply]
I understand.
Thank you.
